qa/L0_perf_deeprecommender/run_test.sh

```diff
@@ -80,20 +80,28 @@ for STATIC_BATCH in $STATIC_BATCH_SIZES; do
 
             set +e
 
-            # Run the model once to warm up. Some frameworks do
-            # optimization on the first requests.  Must warmup similar
-            # to actual run so that all instances are ready
-            $PERF_CLIENT -v -i ${PERF_CLIENT_PROTOCOL} -m $MODEL_NAME -p5000 \
-                         -b${STATIC_BATCH} --concurrency-range ${CONCURRENCY}
-
-            $PERF_CLIENT -v -i ${PERF_CLIENT_PROTOCOL} -m $MODEL_NAME -p5000 \
-                         -b${STATIC_BATCH} --concurrency-range ${CONCURRENCY} \
-                         -f ${NAME}.csv 2>&1 | tee ${NAME}.log
-            if (( $? != 0 )); then
-                RET=1
-            fi
-            curl localhost:8002/metrics -o ${NAME}.metrics >> ${NAME}.log 2>&1
+            # Run test until it passes, or hits max tries. 
```

Comment:
Why not use [--max-trials ](https://github.com/triton-inference-server/client/blob/main/src/c%2B%2B/perf_analyzer/command_line_parser.cc#L357) PA flag?

Reply:
I wasn't aware of this flag, nice call - I will try this.